### DISCUSSION TITLE
Using flowbite with Fresh

### DISCUSSION THREAD

**Discussion by terion-labs** [freshframework/fresh#1363] [Q&A]
In #606 people figured out how to make work headless-ui with fresh via importing like that: 
```ts
import { Switch } from "https://esm.sh/@headlessui/react@1.7.14?alias=react:preact/compat,react-dom:preact/compat,@types/react:preact/compat&external=preact/compat";
```

But the same with flowbite doesn't work:
```ts
import { Button } from 'https://esm.sh/flowbite-react@v0.4.9?alias=react:preact/compat,react-dom:preact/compat,@types/react:preact/compat&external=preact/compat';
```

It doesn't render anything at all and doesn't throw any errors. Just nothing.

Any ideas how to make it work?

**Comment by marvinhagemeister** [comment]
Took a look at it and it seems like we're hitting a bug in `esm.sh` here. With the first link `react` is correctly aliased to `preact`. With the second link that is not the case and `esm.sh` seems to ignore the alias settings. Probably worth it to reach out the folks of `esm.sh` here to make them aware of this issue: https://github.com/esm-dev/esm.sh

**Reply by marvinhagemeister** [reply]
@terion-name The issue was resolved in `esm.sh` (see https://github.com/esm-dev/esm.sh/issues/671) and the provided snippet works now.

**Reply by terion-labs** [reply]
sorry for late response, very tight schedule :(

I've made this:
```
% deno cache --reload main.ts
% deno cache --reload dev.ts 
% deno task start   
```

but nothing changed
what do I need to do to make changes effect?
